src/modules/Kdump.rb

```diff
@@ -966,7 +972,7 @@ def Import(settings)
     #
     # @return [Boolean] is supported
     def fadump_supported?
```

Comment:
Since I got 2 comments about that and the method is actually not published, I will do the change.